### DISCUSSION TITLE
invalid SMP CPUs

### DISCUSSION THREAD

**Discussion by vw117** [utmapp/UTM#6134] [UTM Help (macOS)]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
every time i try to boot a mac 0s 9.2.2 vm after making one , it crashes with this message 
"QEMU error: QEMU exited from an error: qemu-ppc-softmmu: Invalid SMP CPUs 4. The max CPUs supported by machine 'mac99' is 1"

**Configuration**  
* UTM Version: 4.4.5
* macOS Version: sonoma 14.1.2 
* Mac Chip (Intel, M1, ...): M1

**Debug log**  
[crash log.docx](https://github.com/utmapp/UTM/files/13938871/crash.log.docx)

**Comment by osy** [comment]
You can manually set the CPU core count to 1 in system settings.
